src/index.ts

```diff
@@ -414,7 +414,7 @@ export class NodeSSH {
       invariant(typeof parameters[i] === 'string', `parameters[${i}] must be a valid string`)
     }
 
-    const completeCommand = `${command} ${shellEscape(parameters)}`
+    const completeCommand = `${command} ${shellEscape(parameters)}`.trim();
```

Comment:
Instead of `.trim` at the end, let's take an alternative route, and do something like the following
```
`${command}${parameters.length ? ` ${shellEscape(parameters)}` : ''}`
```

Reply:
Okay I pushed alternative way like You've suggested.
Thank You